rocky/reports/views/base.py

```diff
@@ -92,6 +92,16 @@ def setup(self, request, *args, **kwargs):
             get_plugins_for_report_ids(report_ids)
         )
 
+    def check_oois_selection(self):
+        if "all" in self.selected_oois and len(self.selected_oois) > 1:
+            messages.warning(
+                self.request,
+                _(
+                    "You have selected all OOIs and also you've also selected some OOIs from the list. "
+                    "The selected OOIS will be overridden and all OOIs are selected."
+                ),
+            )
```

Comment:
I think it's good to have the warning, but it seems a bit strange to me that this is possible in the first place. In my experience (with web apps), usually the "all" checkbox is unchecked if any of the other checkboxes are deselected; or there's an "all" button that brings you to the next view.

Reply:
This is removed